src/doctopus/web.clj

```diff
@@ -96,6 +97,20 @@
   [_]
   (serve-html (templates/add-tentacle doctopus)))
 
+(defn serve-search-results
```

Comment:
Might be time to start moving a lot of our "serve this or that" functions in to their own NS